docs: make “etc.” spelling consistent

j9t opened the pull request:
Signed-off-by: Jens Oliver Meiert <jens@meiert.com>


<!--
    This comment and the below content is programatically generated.
    You may add a comma-separated list of anchors you'd like a
    direct link to below (e.g. #idl-serializers, #idl-sequence):

    Don't remove this comment or modify anything below this line.
    If you don't want a preview generated for this pull request,
    just replace the whole of this comment's content by "no preview"
    and remove what's below.
-->
***
<a href="https://whatpr.org/html/7102/form-control-infrastructure.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/form-control-infrastructure.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/form-control-infrastructure.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/history.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/history.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/history.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/infrastructure.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/infrastructure.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/infrastructure.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/media.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/media.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/media.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/microdata.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/microdata.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/microdata.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/server-sent-events.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/server-sent-events.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/server-sent-events.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/text-level-semantics.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/text-level-semantics.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/text-level-semantics.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/web-messaging.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/web-messaging.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/web-messaging.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )
<a href="https://whatpr.org/html/7102/window-object.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">/window-object.html</a>  ( <a href="https://whatpr.org/html/7102/f1d1d4a...19b0b72/window-object.html" title="Last updated on Sep 30, 2021, 7:33 PM UTC (19b0b72)">diff</a> )

j9t commented:
@annevk, likewise, how does this one look to you? (Hope okay to ping you about this directly.)

annevk commented on source:
```diff
@@ -20606,7 +20606,7 @@ merger with Demo Group.&lt;/p></code></pre>
     a legal case report,
     a computer program, <!-- "a software package" ? -->
-  etc). This can be a work that is being quoted or <span>referenced</span> in detail (i.e. a
+  etc.). This can be a work that is being quoted or <span>referenced</span> in detail (i.e. a
```
i.e.,*

j9t replied:
Sure, done.

annevk reviewed (commented):
Wanna add the comma while here?

j9t commented:
Updated. (Also added a note to follow up later on the topic of consistency wrt “i.e.” and “e.g.”—will file an issue for this and may volunteer to fix that in the whole spec.)
